### PR TITLE
Joining branches

### PR DESCRIPTION
**Pull request by Apo1lyon**

### CODE REVIEW

File: .obsidian/workspace.json [hunk 1]
```diff
@@ -169,7 +167,8 @@
       }
     ],
     "direction": "horizontal",
-    "width": 255.5
```
Comment (Apo1lyon) [inline .obsidian/workspace.json:172]: Default homepage

File: .obsidian/workspace.json [hunk 2]
```diff
@@ -184,9 +183,14 @@
       "obsidian-full-calendar:Open Full Calendar": false
     }
   },
-  "active": "2dce972683a8dd02",
```
Comment (Apo1lyon) [inline .obsidian/workspace.json:187]: Default homepage

File: .obsidian/workspace.json [hunk 3]
```diff
@@ -223,11 +227,6 @@
     "STUDENT/Система/Шаблоны/Список 306 группа.md",
     "STUDENT/Система/Шаблоны/Список дежурств, 206.md",
     "STUDENT/Система/Шаблоны/Список 206 группы.md",
-    "STUDENT/Система/Шаблоны/Расписание на день.md",
-    "STUDENT/Система/Шаблоны/Лекция.md",
-    "STUDENT/Система/Шаблоны/Краткий синопсис к предмету.md",
-    "STUDENT/Система/Шаблоны/Задание.md",
-    "STUDENT/Система/Шаблоны/Задание. Самостоятельная работа.md",
```
Comment (Apo1lyon) [inline .obsidian/workspace.json:230]: Default homepage

**Review by Apo1lyon — COMMENTED** [review]
Завершенно

**Review by BotAdmin-coderITSE — APPROVED** [review]

**Pull request merged**